build.gradle

```diff
@@ -162,11 +161,13 @@ configure(projectsWithFlags('java')) {
         options.compilerArgs += ['--release', '8']
     }
 
-    tasks.withType(KotlinCompile) {
-        kotlinOptions {
-            jvmTarget = "1.8"
-            freeCompilerArgs = ["-Xjsr305=strict"]
-        }
+    // Configure default duplicates strategy for such as:
+    // - c.l.armeria.versions.properties
+    // - thrift/cassandra.json
+    // - and so on
+    tasks.sourcesJar.duplicatesStrategy = DuplicatesStrategy.EXCLUDE
+    tasks.withType(Copy) {
+        duplicatesStrategy = DuplicatesStrategy.EXCLUDE
```

Comment:
So, should we use `INCLUDE` here?

Reply:
That makes sense. A file could be updated on purpose.